app/assets/javascripts/velumMinionPoller.js

```diff
@@ -0,0 +1,123 @@
+// the semi-colon before function invocation is a safety net against concatenated
+// scripts and/or other plugins which may not be closed properly.
+;( function( $, window, document, undefined ) {
+
+    "use strict";
+
+        // undefined is used here as the undefined global variable in ECMAScript 3 is
```

Comment:
Moreover, lots of these comments looks like they provide from somewhere else. Could you remove them, or trim as much as possible ? (while adding a comment giving credits)